ssdc-rm-common-entity-model/src/main/java/uk/gov/ons/ssdc/common/model/entity/ActionRule.java

```diff
@@ -51,6 +52,10 @@
   @ManyToOne(optional = false)
   private CollectionExercise collectionExercise;
 
+  @Type(type = "jsonb")
+  @Column(columnDefinition = "jsonb")
+  private Map<String, String> uacMetadata;
```

Comment:
Should be `Object` or `String` to allow future flexibility beyond flat key:value pairs of strings